docs/migration-guides/1.3-1.4.md

```diff
@@ -0,0 +1,19 @@
+# Migrate from v1.3 to v1.4
+
+>**TIP:** To learn more about upgrading Kyma, read [this](https://kyma-project.io/docs/master/root/kyma/#installation-upgrade-kyma) document.
+
+## Service Management
+### Google Cloud Platform Service Broker addon
+
+As the Google Cloud Platform Service Broker is being [deprecated](https://cloud.google.com/kubernetes-engine/docs/concepts/google-cloud-platform-service-broker), we changed this addon to use different implementation. If you have already installed the Google Cloud Platform Service Broker in Kyma 1.3, you see this addon with two versions.
+If you haven't installed any GCP services in a given Namespace, delete the Google Cloud Platform Service Broker instance in this Namespace.
```

Comment:
```suggestion
If you still haven't installed any GCP services in your Namespace, delete the GCP Service Broker instance in this Namespace.
```

Reply:
Do we need this sentence? We state it again below in "Consider cleaning up...".

Reply:
I think we need this sentence.